doc/news/changes/minor/20240822Munch

```diff
@@ -0,0 +1,5 @@
+New: The new setting OutputStyle::KeepChanged allows
```

Comment:
```suggestion
New: The new setting OutputStyle::KeepOnlyChanged allows
```

Reply:
I switched to `KeepOnlyChanged`, since that was one of the options that I also considered :+1: